### PR TITLE
feat(accordion): accordion manual testing

### PR DESCRIPTION
**Pull request by timrombergjakobsson**

### CODE REVIEW

**Comment by mJarsater** [comment]
Needs a rebase and might need to make sure you have no lint warnings to make it build

File: src/components/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        <div slot="label">{paddingReset ? 'Disable Padding Reset' : 'Enable Padding Reset'}</div>
+      </tds-toggle>
+      <br />
+      <tds-toggle ref={accordionToggleRef} checked={checkboxState} onTdsToggle={handleAccordionToggleChange}>
+        <div slot="label">{checkboxState ? 'Disable Accordion' : 'Enable Accordion'}</div>
+      </tds-toggle>
+      <h2>Icon Position</h2>
+      <div>
+      <label>
+          <input
```
[the remark this answers is not in the document]
Comment (mJarsater) [reply src/components/Accordion.tsx:91]: Alright, did you try it the way we recommend in the documentation? https://tegel-storybook.netlify.app/?path=/story/intro-events--page

Not, onTdsChange, but rather:
ref={element => element.?addEventListener('tdsChange', (event) )=> {
}}

[the remark this answers is not in the document]
Comment (timrombergjakobsson) [reply src/components/Accordion.tsx:91]: I think I did actually, but will have a look again!

**Comment by mJarsater** [comment]
Could you sort out the lint warnings that causing the build to fail, or do you want someone else to have a look?

**Comment by timrombergjakobsson** [comment]
> Could you sort out the lint warnings that causing the build to fail, or do you want someone else to have a look?

Have not had the time to even look at it ;). But will fix it. As soon as I have the time. All meetings today!

**Comment by mJarsater** [comment]
> ad the tim

Lovelydubly! <3

**Comment by mJarsater** [comment]
> > Could you sort out the lint warnings that causing the build to fail, or do you want someone else to have a look?
> 
> Have not had the time to even look at it ;). But will fix it. As soon as I have the time. All meetings today!

Lovelydubly! <3

**Comment by timrombergjakobsson** [comment]
> > > Could you sort out the lint warnings that causing the build to fail, or do you want someone else to have a look?
> > 
> > 
> > Have not had the time to even look at it ;). But will fix it. As soon as I have the time. All meetings today!
> 
> Lovelydubly! <3

Should be fixed now, will have a look at substituting radio buttons for sdds one instead now!

File: src/components/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+          disabled={!checkboxState}
+          padding-reset={paddingReset}
+          onTdsToggle={() => handleAccordionItemClick('item3')}
+        >
+          This is the panel, which contains associated information with the header. Usually, it contains text, set in the same size as the header. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis laoreet vestibulum fermentum.
+        </tds-accordion-item>
+      </tds-accordion>
+
+      <h2>Accordion with Secondary Mode</h2>
+
```
Comment (mJarsater) [inline src/components/Accordion.tsx:151]: Do I understand correctly that both accordions items right now are being opened/closed with the handleAccordionItemClick function. If so, I think it makes sense to keep one of them just using the accordions own functionality for opeing/closing.

File: src/components/Accordion.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (timrombergjakobsson) [reply src/components/Accordion.tsx]: Got it to work now! Not sure what I did wrong the last time.

**Review by mJarsater — APPROVED** [review]

**Pull request merged**